package.json

```diff
@@ -32,13 +32,11 @@
         "prettier:fix": "npx prettier --write .",
         "prepublishOnly": "npm run release",
         "test:coverage": "npm run build:webpack:dev && jest -c jest.unit.config.js --coverage",
-        "integ:local:chrome": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe chrome --app \"http-server ./build/dev -s\"",
-        "integ:local:chrome:headless": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe chrome:headless --app \"http-server ./build/dev -s\"",
-        "integ:local:firefox": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe firefox --app \"http-server ./build/dev -s\"",
-        "integ:local:firefox:headless": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe firefox:headless --app \"http-server ./build/dev -s\"",
-        "integ:local:safari": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe safari --app \"http-server ./build/dev -s\"",
-        "integ:local:edge": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe edge --app \"http-server ./build/dev -s\"",
-        "integ:local:ie": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe ie --app \"http-server ./build/dev -s\" src/__integ__/unsupportedBrowser.test.ts",
+        "preinteg:browser": "npm run build:webpack:dev",
+        "integ:browser": "testcafe --config-file=./.testcafe.browser.js --app \"http-server ./build/dev -s\"",
```

Comment:
Is `NODE_OPTIONS=--max-http-header-size=80000` no longer required?

Reply:
It's an env variable added in the ci yaml file

No need to manipulate node options locally if not needed (otherwise can update local env)

Reply:
that PR was forked from this one, but there is nothing preventing you from doing:

```npm run integ:browser -- chrome```

which will run tests in chrome :-)

Reply:
We can add to README if youd like :-)

Reply:
I think using configs is good practice, but we can still maintain individual commands for different browsers without having to use a new command:

```
"preinteg:browser": "npm run build:webpack:dev",
"preinteg:headless": "npm run build:webpack:dev",
"integ:browser": "testcafe --config-file=./.testcafe.browser.js --app \"http-server ./build/dev -s\"",
"integ:headless": "testcafe --config-file=./.testcafe.headless.js --app \"http-server ./build/dev -s\"",
"integ:local:chrome": "npm run integ:headless -- chrome",
"integ:local:firefox": "npm run integ:browser -- firefox",
"integ:local:firefox:headless": "npm run integ:headless -- firefox",
```

This still maintains the idea of using configs, and we reuse `preinteg` commands without having to add them into the existing commands for separate browsers. On the other hand, if we will be migrating into a single test framework in the near future, is it still worth to integrate change now?

What are your thoughts, qhanam@ and maniator@?

Reply:
If the `npm run integ:browser -- chrome` works that lgtm.

Reply:
agreed